README.md

```diff
@@ -2,135 +2,137 @@
 ![install two dependencies](https://cloud.githubusercontent.com/assets/8445924/10285715/a0e85302-6b79-11e5-904f-051929fe472b.gif)
 
-After installation we can have a look for vendored or non-vendored dependencies
-by using flag `-Q`. At this moment we should see git commits around two already
-vendored dependencies (`zhash` and `blackfriday`).
+After installation we can have a look for vendored and non-vendored
+dependencies by using flag `-Q`. After previous step we should see git commits
+along with two already vendored dependencies (`zhash` and `blackfriday`):
 
 ![query after install](https://cloud.githubusercontent.com/assets/8445924/10285719/a39282e4-6b79-11e5-8877-7fba19e0d8c0.gif)
 
 Let's install submodules for remaining dependencies, go the limit! Just run
 **manul** with flag `-I` without specifying any dependencies, manul will
-install all detected dependencies with skipping already-vendored.
+install all detected dependencies with skipping already vendored:
 
 ![install all dependencies](https://cloud.githubusercontent.com/assets/8445924/10285722/a63d1e6e-6b79-11e5-9f1e-1e606f3819dc.gif)
 
-Wow, it was crazy. Now update some vendored dependencies, for example,
-`docopt-go` package, by running manul with flag `-U` and specifying import path
-(`github.com/docopt/docopt-go`).
+Wow, it was crazy. Now, to update some vendored dependencies, for example,
+`docopt-go` package, manul should be invoked with flag `-U` and import path
+(`github.com/docopt/docopt-go`):
 
 ![update docopt](https://cloud.githubusercontent.com/assets/8445924/10285723/a8ce9f18-6b79-11e5-87ef-2caca393328c.gif)
 
-**manul** can remove specified submodules of vendored dependencies using flag
-`-R` (remove) and specifying dependencies import path.
+**manul** can be used to remove specified submodules of vendored dependencies
+by using flag `-R` (remove) and specifying dependencies import path:
 
 ![removing](https://cloud.githubusercontent.com/assets/8445924/10285727/ab587b50-6b79-11e5-9b5b-b7c7ff264506.gif)
 
-
 By the way, manul can detect and remove unused vendored dependencies using `-C`
-(clean) flag.
+(clean) flag:
 
 ![unused dependencies](https://cloud.githubusercontent.com/assets/8445924/10285731/ae1d0270-6b79-11e5-9e97-151b7d77402a.gif)
 
-Let's summarize.
+Let's summarize:
 
-- `-I [<dependency>...]` - install git submodules for specified or all dependencies
-- `-U [<dependency>...]` - update specified or all already-vendored dependencies.
-- `-R [<dependency>...]` - remove git submodules of specified or all
-    dependencies.
-- `-Q [<dependency>...]` - list all used dependencies.
-- `-C` - detect and remove all git submodules of unused vendored dependencies.
+- `-I [<dependency>...]` - install git submodules for specified/all dependencies;
+- `-U [<dependency>...]` - update specified/all already vendored dependencies;
+- `-R [<dependency>...]` - remove git submodules for specified/all dependencies.
+- `-Q [<dependency>...]` - list all imported dependencies.
```

Comment:
what is it mean `imported dependencies`? I think `used dependencies` it's more obvious